contact.html

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
 <html lang = "en">
+	<!-- http://www.w3schools.com/html/html_layout.asp -->
+	
   <title>
   Poodle Security Training School | Contact Us
   </title>
-<body style="font-family:verdana;">
-<!-- http://www.w3schools.com/html/html_layout.asp -->
-<center>
-	<header><h1>Poodle Security Training School</h1></header>
+  <link rel="shortcut icon" href="http://www.hddevelop.com/images/pawprint.ico">
+  <meta charset="utf-8">
+	
+	
+	<style>
+	body{
```

Comment:
setting all text within <body> tags to be in Verdana, or a sans serif font if Verdana is not available